### PR TITLE
Generate alloy/src/main/resources/META-INF/aui.tld

### PR DESCRIPTION
**Pull request by stiemannkj1**

### CODE REVIEW

File: alloy/src/main/resources/META-INF/aui.tld [hunk 1]
```diff
@@ -3,7 +3,7 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-jsptaglibrary_2_1.xsd"
         version="2.1">
-   <tlib-version>3.0.5</tlib-version>
+   <tlib-version>3.0.6</tlib-version>
```
Comment (stiemannkj1) [inline alloy/src/main/resources/META-INF/aui.tld]: Seems that the taglib2tld.xsl is generating the wrong version in this branch `3.0.x-legacy`.

This line is the issue:
https://github.com/liferay/liferay-faces/blob/master/alloy/src/main/resources/META-INF/facelets2tld.xsl#L11

@ngriffin7a, should I fix this or is it by-design?

Comment (ngriffin7a) [inline alloy/src/main/resources/META-INF/aui.tld]: @stiemannkj1 Please fix if <=15m of your time otherwise don't worry about it

**Comment by ngriffin7a** [comment]
Merged in to 3.0.x-legacy. Thank you.

File: alloy/src/main/resources/META-INF/aui.tld (hunk 1 above)
Comment (stiemannkj1) [inline alloy/src/main/resources/META-INF/aui.tld]: Done in #880 and #881.

**Pull request closed without merging**